### PR TITLE
Feat/repeater field

### PR DESCRIPTION
**Pull request by jwinding**
## Feature description
Adds a repeater field component, for entering multiple copies of the same input. 

## Solution description
Just adds a new input component. Changes to the behaviour of dateTimePicker which now opens a modal. This is ugly, and needs a rework, but it was done this way because the previous dateTimePicker simply did not work inside any kind of container. 

The new component also needs more styling, only basic UX is implemented.

Uses the fieldset component by Teddy, as added by another PR, with minimal changes. This is meant to be merged after the Stepper-Poc+navigation button PR. 

## What areas is affected by these changes?.
Datepicker component behavior.

## Is there any existing behaviour change of other features due to this code change?
Datepicker.

## Covered unit tests cases / E2E test cases?
No.

## Are your code structured in a way so that reviewers can understand it?
Yes. Check the prop interface and the story to see how to use the component. 

## Was this feature tested in the following environments?
- [x] On a iOS device/simulator.

### CODE REVIEW

**Review by dannil76 — COMMENTED** [review]
Have opinion on this line:
_Uses the fieldset component by Teddy, as added by another PR, with minimal changes. This is meant to be merged after the Stepper-Poc+navigation button PR._

Should avoid refering to stuff one can't relate to. This requires that you have the full history insight of the code worked on.

**Review by TeddyAndersson — CHANGES_REQUESTED** [review]
This branch can't be tested due to that ButtonField.js no longer exists. This causes the application to crash when you try to run Storybook.

**Comment by TeddyAndersson** [comment]
This branch can't be tested due to that ButtonField.js no longer exists. This causes the application to 

> Have opinion on this line: (Solution description)
> _Uses the fieldset component by Teddy, as added by another PR, with minimal changes. This is meant to be merged after the Stepper-Poc+navigation button PR._
> 
> Should avoid refering to stuff one can't relate to. This requires that you have the full history insight of the code worked on.
Agree with Dan on this one it's hard for someone without knowledge of this use case to understand the context.

I think referring is fine as long as there is links to the PRs so that one can checkout the code that is related (y)

**Comment by jwinding** [comment]
I've now rebased this on top of develop, and fixed some issues with it. The repeater field should now be testable by going into the storybook and testing the repeater field story.

Regarding the FieldSet component, that is now in dev, so I think it's even more okay for this branch to use it.

File: source/components/molecules/FormField/FormField.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Input, FieldLabel, Select, Text } from 'source/components/atoms';
+import { Input, FieldLabel, Select, Text, Button } from 'source/components/atoms';
```
Comment (TeddyAndersson) [inline source/components/molecules/FormField/FormField.js:3]: Button import is unused in this file, remove it from imports. The same thing goes for SubStepButton further down.

**Review by TeddyAndersson — CHANGES_REQUESTED** [review]

File: source/components/molecules/FormField/FormField.js (hunk 1 above)
Comment (jwinding) [reply source/components/molecules/FormField/FormField.js]: Fixed.

**Review by TeddyAndersson — APPROVED** [review]
Looks good to me now

**Pull request merged**